api/src/redis_cache.rs

```diff
@@ -22,13 +22,22 @@ pub async fn warm_validators(
 ) -> anyhow::Result<()> {
     info!("Loading validators from DB");
     let warmup_timer = Instant::now();
-    let validators = store::utils::load_validators(
+
+    let bonds = store::utils::fetch_bonds(&context.read().await.bonds_url).await?;
+    let mut validators = store::utils::load_validators(
         &context.read().await.psql_client,
         scoring_url,
         DEFAULT_EPOCHS,
         DEFAULT_COMPUTING_EPOCHS,
     )
     .await?;
+
+    for bond in bonds {
+        if let Some(validator) = validators.get_mut(&bond.vote_account) {
+            validator.self_stake += bond.effective_amount;
+        }
+    }
+
     let validators_json = serde_json::to_string(&validators).unwrap();
```

Comment:
Would not be usable to add a value `bond_effective/funded_amount` into the validator record to be part of the json and in case of some investigation we can find what the `self_stake` was originally by reducing the amount by the value taken from the bond amount value (either it is used effective or funded based of my point from prior comment).

Reply:
I think this requires a rework - maybe in the future we can add this / after launch